spec/shell-unix-bash-spec.coffee

```diff
@@ -60,6 +60,48 @@ describe "Shell script grammar", ->
       expect(tokens[1][0]).toEqual value: 'lorem ipsum', scopes: ['source.shell', 'meta.herestring.shell', scope]
       expect(tokens[1][1]).toEqual value: delim, scopes: ['source.shell', 'meta.herestring.shell', scope, 'punctuation.definition.string.end.shell']
 
+  it "tokenizes heredocs", ->
+    delimsByScope =
+      "ruby": "RUBY"
+      "python": "PYTHON"
+      "applescript": "APPLESCRIPT"
+      "shell": "BASH"
```

Comment:
Looks like this object has duplicate `shell` keys meaning only `SHELL` is used since the last value wins when duplicate keys exist.